instruments/thorlabs/thorlabsapt.py

```diff
@@ -1169,8 +1169,8 @@ class MotorChannel(ThorLabsAPT.APTChannel):
                 # TODO: Z8xx and Z6xx models. Need to add regex support to motor models, too.
                 "PRM1-Z8": (
                     u.Quantity(1919.64, "count/deg"),
-                    NotImplemented,
-                    NotImplemented,
+                    u.Quantity(42941.66, u.sec / u.deg),
+                    u.Quantity(14.66, u.sec ** 2 / u.deg),
```

Comment:
Weird thing here: Distances are incorporated as "encouder counts per degree", however, velocity and acceleration according to the Thorlabs manual table as scaling factors... It's also not counts per second, so leaving velocity units away.